src/saving/InProgressLoad.cs

```diff
@@ -49,6 +49,11 @@ public void ReportStatus(bool success, string message, string exception = "")
         this.exception = exception;
     }
 
+    public static bool CheckIsLoading()
```

Comment:
This should be replaced with a static property. That way there is no need to have a separate backing variable, as long as the set is private so that it isn't unnecessarily exposed.